media-sound/wavpack/wavpack-5.0.0.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="Hybrid lossless audio compression format & tools"
+DESCRIPTION="WavPack is a completely open audio compression format, providing \
+lossless, high-quality lossy, and a unique hybrid compression mode."
+HOMEPAGE="http://wavpack.com/"
+COPYRIGHT="1998-2016 David Bryant"
+LICENSE="WavPack"
+REVISION="1"
+SOURCE_URI="http://wavpack.com/wavpack-$portVersion.tar.bz2"
+CHECKSUM_SHA256="918d7e32a19598df543b17fff840b10a0880f87296f9e32af454d256b6a64049"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	wavpack$secondaryArchSuffix = $portVersion
+	lib:libwavpack$secondaryArchSuffix = 1.2.0 compat >= 1
+	cmd:wavpack = $portVersion
+	cmd:wvgain = $portVersion
+	cmd:wvunpack = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	wavpack${secondaryArchSuffix}_devel
+	devel:libwavpack$secondaryArchSuffix = 1.2.0 compat >= 1
+	"
+REQUIRES_devel="
+	wavpack$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libiconv$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:diff
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -i -e "2s|\=.*$|\= $includeDir/wavpack|" \
```

Comment:
I would recommend replacing
```
	sed -i -e "2s|\=.*$|\= $includeDir/wavpack|" \
```
by
```
	sed -i -e "/wpincludedir/ \
		s|/include\(/wavpack\)|/$relativeOldIncludeDir\1|" \
```
because this sed command is a little bit safer, in case the wpincludedir is moved to another line in a future release.